de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
I can see how this could be confusing though for the users.  I believe the same thing happens if you remove a user from a list - anything shared with that list will no longer be shared with that user because they're no longer part of the list.  Do we need to rethink this?